Samples/iOS-Swift/iOS-Swift/TraceTestViewController.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import Sentry
+import UIKit
+
+class TraceTestViewController: UIViewController {
+    
+    var loadSpan: Span?
+    @IBOutlet weak var imageView: UIImageView!
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        initialize()
+    }
+    
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+        initialize()
+    }
+        
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    private func initialize() {
+        //Start a transaction to determine how long it take to display the view.
+        loadSpan = SentrySDK.startTransaction(name: "TraceTestViewController", operation: "UI")
```

Comment:
what's about using the Static API and setting it to the Scope instead of a global field? or isn't possible yet?
eg:

```
Sentry.configureScope(scope -> { scope.span = span})
val span = Sentry.getSpan()
```

Reply:
No span in the scope yet.